diff -urpN b/arch/x86/kernel/msr.c a/arch/x86/kernel/msr.c
--- b/arch/x86/kernel/msr.c	2023-06-05 08:29:47.000000000 +0100
+++ a/arch/x86/kernel/msr.c	2023-06-06 13:11:56.199655365 +0100
@@ -116,9 +116,9 @@ static ssize_t msr_write(struct file *fi
 	int err = 0;
 	ssize_t bytes = 0;
 
-	err = security_locked_down(LOCKDOWN_MSR);
+	/*err = security_locked_down(LOCKDOWN_MSR);
 	if (err)
-		return err;
+		return err;*/
 
 	err = filter_write(reg);
 	if (err)
@@ -179,9 +179,9 @@ static long msr_ioctl(struct file *file,
 			err = -EFAULT;
 			break;
 		}
-		err = security_locked_down(LOCKDOWN_MSR);
+		/*err = security_locked_down(LOCKDOWN_MSR);
 		if (err)
-			break;
+			break;*/
 
 		err = filter_write(regs[1]);
 		if (err)
